src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -196,52 +201,36 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
         });
     }
 
-    /**
-     * Sucess callback for classification change
-     *
-     * @private
-     * @return {void}
-     */
-    classifiationChangeSuccessCallback = (file: BoxItem) => {
+    fetchClassificationSuccessCallback = (classificationInfo: any): void => {
```

Comment:
Can you type this?

Reply:
and add jsdocs

Reply:
Yup, wanted to get structure down before adding new types (If I need to) and handling via onError